back-end/src/database/migrations/20240412190405__VERSION_06.ts

```diff
@@ -0,0 +1,19 @@
+import type { Knex } from 'knex';
+
+export async function up(knex: Knex): Promise<void> {
+	await knex.schema.dropTableIfExists('saves');
+
+	await knex.schema.createTable('saves', (table) => {
```

Comment:
Would be nice to populate the database so that there would be mock-data in presentation